examples/layer-browser/package.json

```diff
@@ -9,8 +9,8 @@
     "start-local-production": "webpack-dev-server --env.local --env.production --progress --hot --open"
   },
   "dependencies": {
-    "@loaders.gl/core": ">=1.0.0-alpha.2",
-    "@loaders.gl/ply": ">=1.0.0-alpha.2",
+    "@loaders.gl/core": "^1.0.0-alpha.4",
+    "@loaders.gl/ply": "^1.0.0-alpha.4",
     "@luma.gl/addons": "^7.0.0-beta.4",
```

Comment:
Update 
- loaders to `^1.0.0` 
- luma to `7.0.0-rc.1`!

Reply:
I am thinking to make an script to upgrade these.... We have more and more modules now

Reply:
Theoretically we won't have to upgrade these again until loaders.gl v2.